downstream/modules/platform/proc-account-linking.adoc

```diff
@@ -45,3 +45,5 @@ If you log into {PlatformNameShort} for the first time and are prompted to chang
 
 .A diagram of the account linking flow
 image:account-linking-flow.png[Account linking flow]
+
+After you have migrated your user account, you can manage your account from the btn:[Access Management] menu. See link:{URLCentralAuth}/gw-managing-access[Managing access with role based access control]. 
```

Comment:
Menus should be coded as menu:Access Management[] and not btn:

This will be fixed in a separate PR.